components/MapResults.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <GMap
+    ref="gMap"
+    map-type-id="terrain"
+    language="en"
+    :cluster="{ options: { styles: clusterStyle } }"
+    :center="{ lat: currentLocation.lat, lng: currentLocation.lng }"
+    :options="options"
+    :zoom="2"
+  >
+    <GMapMarker
+      v-for="location in locations"
+      :key="location.id"
+      :position="{ lat: location.lat, lng: location.lng }"
```

Comment:
what's the difference between location and currentLocation? might be good to better describe `location`.